cli/command/container/opts.go

```diff
@@ -297,6 +299,10 @@ func addFlags(flags *pflag.FlagSet) *containerOptions {
 
 	flags.BoolVar(&copts.init, "init", false, "Run an init inside the container that forwards signals and reaps processes")
 	flags.SetAnnotation("init", "version", []string{"1.25"})
+
+	flags.Var(copts.annotations, "annotation", "Add annotations to container")
```

Comment:
Something like this, maybe?

```suggestion
	flags.Var(copts.annotations, "annotation", "Add annotations to the container (passed through to the OCI runtime)")
```

(for the relevant item in https://github.com/docker/cli/pull/4156#issuecomment-1507468378)

Reply:
^^ LGTM (forgot to write that down, but we discussed that as description for the flag) (thanks!)

Reply:
done